### PR TITLE
[civetweb] Add optional SSL support

### PR DESCRIPTION
**Pull request by gjasny**

### CODE REVIEW

**Comment by gjasny** [comment]
The CI error seems to be unrelated and possibly originates from the `spdk-ipsec:x64-linux` port.

**Comment by PhoebeHui** [comment]
@gjasny, it's unreated, please ignore it now, we will take a look at the baseline issue.

File: ports/civetweb/CONTROL [hunk 1]
```diff
@@ -1,4 +1,8 @@
 Source: civetweb
 Version: 2019-07-05
```
Comment (PhoebeHui) [inline ports/civetweb/CONTROL]: Could you also update the Version?

File: ports/civetweb/CONTROL [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
 Description: Easy to use, powerful, C/C++ embeddable web server.
-Supports: !uwp
\ No newline at end of file
+Supports: !uwp
+
+Feature: ssl
```
Comment (PhoebeHui) [inline ports/civetweb/CONTROL]: Have you test the feature locally?

**Review by PhoebeHui — CHANGES_REQUESTED** [review]

File: ports/civetweb/CONTROL (hunk 2 above)
Comment (gjasny) [reply ports/civetweb/CONTROL]: Yes, on macOS.

File: ports/civetweb/CONTROL (hunk 1 above)
Comment (gjasny) [reply ports/civetweb/CONTROL]: Added `-2`.

Comment (PhoebeHui) [reply ports/civetweb/CONTROL]: Could you update it to "2019-07-05-1"?

**Comment by PhoebeHui** [comment]
Test the feature 'ssl' locally, it test pass with x86-windows x64-windows, and Linux triplets.

File: ports/civetweb/CONTROL (hunk 1 above)
Comment (gjasny) [reply ports/civetweb/CONTROL]: Sure. I was interpreting [control-files](https://vcpkg.readthedocs.io/en/latest/maintainers/control-files/) as if I should add a `-2`

> Then if you update the source version today, you should give it version 2019-06-01. If you need to make a change which doesn't adjust the source version, you should give it version 2019-02-14-2.

**Review by PhoebeHui — APPROVED** [review]

**Comment by dan-shaw** [comment]
@gjasny thanks for the PR!

**Pull request merged**